### PR TITLE
Add volt-watt controller

### PR DESCRIPTION
**Pull request by daniel-thom**

### CODE REVIEW

File: disco/pydss/config/pv_controllers.toml [hunk 1]
```diff
@@ -50,3 +50,16 @@ percent_p_cutout = 10
...
+Control2 = "None"
+Control3 = "None"
+DampCoef = 0.8
+PFlim = 0.9
+PminVW = 20
+Priority = "Watt"
+QlimPU = 0.44
+VWtype = "Available Power"
+uMaxC = 1.1
+uMinC = 1.04
```
Comment (daniel-thom) [inline disco/pydss/config/pv_controllers.toml:65]: Is this the correct value?

File: disco/pydss/config/pv_controllers.toml [hunk 2]
```diff
@@ -50,3 +50,16 @@ percent_p_cutout = 10
 Efficiency = 100
 Priority = "Var"
 DampCoef = 0.8
+
+[volt_watt]
```
Comment (daniel-thom) [inline disco/pydss/config/pv_controllers.toml:54]: Do we want a more specific name?

File: disco/pydss/config/pv_controllers.toml (hunk 1 above)
Comment (wenbowangnrel) [reply disco/pydss/config/pv_controllers.toml:65]: Can we change `uminC` to `1.06`? 1.06 is IEEE standard and HECO standard.

File: disco/pydss/config/pv_controllers.toml (hunk 2 above)
Comment (wenbowangnrel) [reply disco/pydss/config/pv_controllers.toml:54]: `volt_watt` looks okay to me.

**Review by wenbowangnrel — APPROVED** [review]
I think they look good, I approve it, and please change the uMinC to `uMinC=1.06` before merge.

**Pull request merged**